stdlib/2/typing.pyi

```diff
@@ -385,7 +385,8 @@ class Pattern(Generic[AnyStr]):
 
 # Functions
 
-def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+def get_type_hints(obj: Callable, globalns: Optional[dict],
```

Comment:
These should both be `: Optional[Dict[Text, Any]] = ...` (the arguments are optional and the keys must be strings).